assets/src/components/RangeSlider.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import 'rc-slider/assets/index.css';
+import 'rc-tooltip/assets/bootstrap.css';
+import {Range} from 'rc-slider';
+
+const rangeSlider = props => {
+    const wrapperStyle = { width: "100%" };
+    const marks = {};
```

Comment:
remove semicolons for statements seem to be unnecessary and they are all over the code in various files.